spec/spec_helper.rb

```diff
@@ -1,4 +1,20 @@
 require 'pathname'
+
+require 'simplecov'
+
+SimpleCov.start do
+  track_files 'lib/**/*.rb'
+  add_filter 'lib/reek/version.rb' # version.rb is loaded too early to test
+end
+
+if ENV['FAIL_ON_LOW_COVERAGE'] == '1'
+  SimpleCov.at_exit do
+    SimpleCov.result.format!
+    SimpleCov.minimum_coverage 98.71
```

Comment:
I'd rather not pollute our source code with "nocov" everywhere.
Especially since I am highly sceptical of simplecov anyways (i'd prefer setting up mutant properly, but thats a different story)